src/decoder/mod.rs

```diff
@@ -1,4 +1,4 @@
-use num_traits::{FromPrimitive, Num};
+use num_traits::{Num};
```

Comment:
Removing the `num_traits` dependency as well should be pretty simple, right? Should only be a simple change in https://github.com/image-rs/image-tiff/blob/master/src/decoder/mod.rs#L194.

Reply:
Huh, right. That function is only used twice.

Reply:
To me it doesn't even seem like it uses the `Num` bound.

Reply:
Right, I thought there was arithmetic going that includes the number type but the `Wrapping` trait has already been extracted.